modeling.py

```diff
@@ -272,7 +272,7 @@ def generate_self_debiasing(self, input_text: str, debiasing_texts: List[str], n
         if min_length is not None:
             min_length = min_length + input_length
         if max_length is not None:
-            max_length = max_length + input_length
+            max_length = min(1024, max_length + input_length)
```

Comment:
Great, thank you! I think it would be safer to replace `1024` with `self.model._model.config.max_position_embeddings` so that this will not break for other language models with a different maximum sequence length.

Reply:
True, I hadn't thought of that. Code has been amended.